js/coinbase.js

```diff
@@ -53,7 +55,10 @@ module.exports = class coinbase extends Exchange {
                 'fetchTime': true,
                 'fetchTrades': undefined,
                 'fetchTransactions': undefined,
+                'fetchTransactionsWithMethod': true,
```

Comment:
not sure if this should be in here

Reply:
It's an exchange-specific helper, so, no, it should not be there.